### ISSUE TITLE
Certain profiles will create a "Begin folder monitoring" bug

### ISSUE THREAD

**Issue by bgq527** [CapPow/HerbASAP#51]
The cause is unsure, but certain created profiles will disallow "Begin folder monitoring" to not work.

Steps is as follows:
1. Select a buggy profile (cause unknown)
2. Click "Begin folder monitoring"
3. The "Begin folder monitoring" button will change to "Stop folder monitoring" for ~0.5 seconds before quickly reverting back to "Begin folder monitoring"

Seems to happen on both 0.2 and 0.1 betas, folders definitely still exist. Folder monitoring with this bug does not work (even if you add items to the folder, etc).

Cause unknown.

See attached GIF:
![issue](https://user-images.githubusercontent.com/37383659/95868762-98ee0f00-0d38-11eb-9318-6cc9e88d496f.gif)

**Comment by CapPow** [comment]
@bgq527, thanks for opening the issues! I'm inclined to wonder if the profile bugs (#53) are related to profiles generated from earlier versions? If you get time can you attempt to reproduce these issues on the latest version after wiping your user stored user preferences?

**Comment by bgq527** [comment]
Yes, I have. I should've noted in my the original post that I've already tried to delete profiles as well as the HerbASAP registry folder.

Additionally, I know that it's not a profile issue nor an operating system issue - a Ubuntu 20.04 install that has never run HerbASAP before exhibits the same problem. See the below GIF of the behavior.

Also, I know that in the below example technician is empty, but it seems to have nothing to do with it. I haven't tried disabling all of the post-processing options (within the setup wizard) to see if that fixes it, maybe I will soon and let you know. Unfortunately, it is still unknown to me the true underlying cause of the problem.

![ubuntu-profile-bug](https://user-images.githubusercontent.com/37383659/98003479-cae51500-1dbc-11eb-976a-f2a1f66e607a.gif)
